redhat-operators/tektoncd-pipeline-operator/tektoncd-pipeline-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,250 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: "Developer Tools"
+    description: An operator to install TektonCD Pipeline
+    containerImage: quay.io/openshift-pipeline/nikthoma_tektoncd-pipeline-operator:v0.2.0-1
+    createdAt: 2019-03-15T19:44:21Z
+    certified: "false"
+    support: Red Hat, Inc.
+    repository: https://github.com/openshift/tektoncd-pipeline-operator
+    alm-examples: |
+      [
+        {
+          "apiVersion": "tekton.dev/v1alpha1",
+          "kind": "Install",
+          "metadata": {
+            "name": "example-install",
+            "namespace": "tekton-pipelines"
+        },
+          "spec": {}
+        }
+      ]
+  name: tektoncd-pipeline-operator.v0.0.1
+  namespace: tekton-pipelines
```

Comment:
nit: Please use `placeholder` for the operator namespace

Reply:
done